mne/source_estimate.py

```diff
@@ -26,7 +26,6 @@
 from .io.base import ToDataFrameMixin, TimeMixin
 from .externals.h5io import read_hdf5, write_hdf5
 
-
```

Comment:
Revert, PEP8 problem

Reply:
@padmasundaram you should use a flake8 linter. On sublime text, you can install the Anaconda package and on Spyder, ask Fahimeh. For vim too, there must be something